administrator/language/de-DE/de-DE.com_joomlaupdate.ini

```
@@ -30,6 +30,9 @@ COM_JOOMLAUPDATE_TOOLBAR_CHECK="Auf Updates prüfen"
 COM_JOOMLAUPDATE_PREUPDATE_HEADING_CHECKED="Überprüft"
 COM_JOOMLAUPDATE_PREUPDATE_HEADING_REQUIREMENT="Vorraussetzung"
 COM_JOOMLAUPDATE_PREUPDATE_UNKNOWN_EXTENSION_MANIFESTCACHE_VERSION="Unbekannte Version"
+COM_JOOMLAUPDATE_PREUPDATE_CHECK_EXTENSION_AUTHOR_URL="Erweiterung Autoren-URL"
+COM_JOOMLAUPDATE_PREUPDATE_CHECK_NOT_COMPLETE="Die Kompatibilitätsprüfungen sind noch nicht abgeschlossen – bitte warten."
```

```suggestion
COM_JOOMLAUPDATE_PREUPDATE_CHECK_NOT_COMPLETE="Die Kompatibilitätsprüfung ist noch nicht abgeschlossen – Bitte warten."
```

Einzahl oder nicht. Ist ja eine große Überprüfung aller Erweiterungen?